app/controllers/items_controller.rb

```diff
@@ -7,4 +7,17 @@ def index
   def show 
     @item = Item.find(params[:id])
   end
+
+  def update
+    @item = Item.find(params[:id])   
+    @merchant = @item.merchant
+    if params[:disable]
+      @item.update(status: params[:disable].to_i)
+      redirect_to merchant_items_path(@merchant.id)
+    end 
+    if params[:enable] 
+      @item.update(status: params[:enable].to_i)
+      redirect_to merchant_items_path(@merchant.id)
```

Comment:
Just curious if the param is not sent as an integer by default or not.

Reply:
It's sent as a string, but the integer is set by the enum in the item model page